src/csharp/Microsoft.Spark/Sql/CustomPicklers.cs

```diff
@@ -40,4 +40,15 @@ public void pickle(object o, Stream outs, Pickler currentPickler)
             currentPickler.save(((Date)o).GetInterval());
         }
     }
+
+    /// <summary>
+    /// Custom pickler for Timestamp objects.
+    /// </summary>
+    internal class TimestampPickler : IObjectPickler
+    {
+        public void pickle(object o, Stream outs, Pickler currentPickler)
```

Comment:
Do we want to rename `outs` to either `outStream` or just `out`? `outs` reads a little funny to me.

Reply:
Looking through some other examples, `outs` has been used to stand for `output stream`.